include/jemalloc/internal/spin_delay_arm.h

```diff
@@ -0,0 +1,30 @@
+#include "jemalloc/internal/jemalloc_preamble.h"
+#include <sys/auxv.h>
+
+/* Define HWCAP_SB if not already defined in system headers */
+#ifndef HWCAP_SB
+#define HWCAP_SB (1UL << 56) /* Speculation Barrier */
```

Comment:
Need `1ULL` for 32-bit builds.
https://app.travis-ci.com/github/jemalloc/jemalloc/jobs/632564590#L757